feat(discover-homepage): Update Discover homepage docs for GA (#5698)
getsentry/sentry-docs: src/docs/product/discover-queries/index.mdx

![Full view of the Discover Homepage with query cards and button to build new queries.](./discover-homepage.png)

> The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage will open the query builder where you can immediately begin building a new query. By default, this will show the pre-built "All Events" query. Columns in the homepage can be [edited and saved, or saved from other queries](#set-as-default). To navigate back to the [query management page](#saved-queries), where pre-built and saved queries can be found, click the "Saved Queries" button.

Reviewer: A couple things:

- I don't understand what you mean by "saved from other queries" and the section on default homepages doesn't really explain this so I've edited it to mirror the wording we used in that section
- I understand putting the link in the words "query management page" but it's confusing to link it to a heading called Saved Queries so I've moved the link.

```suggestion
The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin building a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set it as your default](#set-as-default). To navigate back to the query management page(#saved-queries), where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.
```

Reviewer: Let's do it!

> The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin building a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set it as your default](#set-as-default). To navigate back to the query management page, where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.

Reviewer: ```suggestion
The <SandboxLink scenario="discover" projectSlug="react">Discover</SandboxLink> homepage displays the query builder where you can immediately begin creating a new query. By default, the page shows the pre-built "All Events" query, but you can customize the homepage and [set the custom version as your default](#set-as-default). To navigate back to the query management page, where you can find [pre-built](#pre-built-queries) and [saved](#saved-queries) queries, click the "Saved Queries" button.
```

### Set as Default

When you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.

> You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on saved queries, which you can find by clicking the "Saved Queries" button.

Reviewer: ```suggestion
You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on both saved and pre-built queries, which you can find by clicking the "Saved Queries" button.
```

### Set as Default

When you navigate to **Discover**, the homepage opens the query builder where you can immediately begin creating a new query. By default, the page displays the pre-built "All Events" query.

> You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.

Author: I also want to shim in the whole, you can go to a saved query and also set that as your homepage but I'm not exactly sure how to word it.

> You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. You can also find "Set as Default" in [saved queries](#query-results) as well.

> You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded.

Reviewer: Wondering if you can set other pre-built queries as defaults? Should we be mentioning this?
```suggestion
You can customize your homepage by changing the query, choosing which columns are displayed, and how charts appear, and then clicking the "Set as Default" button. Navigating to **Discover** will now open the query builder with this configuration pre-loaded. The "Set as Default" button is also available on saved queries, which you can find by clicking the "Saved Queries" button.
```

Reviewer: Updated my suggestion

Reviewer: Nevermind
